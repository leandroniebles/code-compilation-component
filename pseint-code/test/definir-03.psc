Proceso prueba
  definir a como caracter;
  definir b como carácter;
  definir c como caracteres;
  definir d como cadena;
  definir e como texto;
  a<-2;
  b<-2;
  c<-2;
  d<-2;
  e<-2;
FinProceso