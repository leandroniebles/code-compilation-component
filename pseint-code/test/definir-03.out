=== Line 7: SynError 125
=== Line 8: SynError 125
=== Line 9: SynError 125
=== Line 10: SynError 125
=== Line 11: SynError 125
